packages/bot-web-ui/src/components/dashboard/bot-builder/toolbar/toolbar.tsx

```diff
@@ -0,0 +1,252 @@
+import { Button, Icon, Popover, Dialog } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import React from 'react';
+import LoadModal from 'Components/load-modal';
+import SaveModal from 'Components/save-modal';
+import { tabs_title } from 'Constants/bot-contents';
+import { popover_zindex } from 'Constants/z-indexes';
+import { connect } from 'Stores/connect';
+import RootStore from 'Stores/index';
+
+interface IconButtonProps {
+    popover_message: string;
+    icon: string;
+    icon_id: string;
+    iconOnClick: () => void;
+    icon_color?: string;
+}
+
+const IconButton = ({ popover_message, icon, icon_id, icon_color, iconOnClick }: IconButtonProps) => (
+    <Popover alignment='bottom' message={popover_message} zIndex={popover_zindex.TOOLBAR} should_disable_pointer_events>
+        <Icon
+            icon={icon}
+            id={icon_id}
+            className='toolbar__icon'
+            onClick={iconOnClick}
+            {...(icon_color ? { color: icon_color } : null)}
+        />
+    </Popover>
+);
+
+interface ToolbarButtonProps {
+    popover_message?: string;
```

Comment:
Do we need popover_message as its not being used in the ToolbarButton component?

Reply:
I think it is required, the popover component inside the ToolbarButton accepts popover_message prop, so its there.